includes/forms.php

```diff
@@ -0,0 +1,640 @@
+<?php
+/**
+ * Forms
+ *
+ * @package    Restrict Content
+ * @subpackage Forms
+ * @copyright  Copyright (c) 2017, Sandhills Development, LLC
+ * @license    GPLv2
+ */
+
+/**
+ * Displays the login form fields
+ *
+ * Based on the requested action, this could be the lost password form,
+ * lost password "check your email" message, change password form,
+ * or the login form template.
+ *
+ * @param array $args Arguments to override the defaults (including redirect URL).
+ *
+ * @return string HTML content
+ */
+function rc_login_form_fields( array $args = array() ) {
+
+	$defaults = array( 'redirect' => rc_get_current_url() );
+
+	$args = wp_parse_args( $args, $defaults );
+
+	if ( ! empty( $_GET['redirect'] ) ) {
+		$args['redirect'] = urldecode( $_GET['redirect'] );
+	}
+
+	ob_start();
+
+	if ( isset($_REQUEST['rc_action']) && $_REQUEST['rc_action'] === "lostpassword") {
+
+		echo rc_lost_password_form();
+
+	} elseif ( isset($_REQUEST['rc_action']) && $_REQUEST['rc_action'] === "lostpassword_checkemail") {
+
+		echo rc_lost_password_check_email_message();
+
+	} elseif ( isset($_REQUEST['rc_action']) && ( $_REQUEST['rc_action'] === "lostpassword_reset" || $_REQUEST['rc_action'] === "reset-password" ) ) {
+
+		echo rc_change_password_form( $args );
+
+	} else {
+
+		if ( ! is_user_logged_in() ) :
+
+			rc_show_error_messages( 'login' );
+
+			if ( isset( $_GET['password-reset'] ) && $_GET['password-reset'] == 'true' ) : ?>
+				<div class="rc-message">
+					<p class="rc-success"><?php _e( 'Password changed successfully. Please log in with your new password.', 'restrict-content' ); ?></p>
+				</div>
+			<?php endif; ?>
+
+			<form id="rc_login_form"  class="<?php esc_attr_e( $args['class'] ); ?>" method="POST" action="<?php echo esc_url( rc_get_current_url() ); ?>">
```

Comment:
I think `esc_attr_e()` is really meant for translated text, so maybe `echo esc_attr()` would be better for the class attribute?